brax/training/agents/ppo/train.py

```diff
@@ -319,6 +278,45 @@ def train(
   gradient_update_fn = gradients.gradient_update_fn(
       loss_fn, optimizer, pmap_axis_name=_PMAP_AXIS_NAME, has_aux=True)
 
+  def random_translate_pixels(obs: Mapping[str, jax.Array], key: PRNGKey):
```

Comment:
`_random_translate_pixels` etc. and should be moved back out for tidiness, sorry if that was unclear :|